xapian-applications/omega/omegatest

```diff
@@ -206,12 +206,27 @@ testcase '' P=text a="Example" b="random"
 
 # Feature tests for $match.
 printf '$match{$cgi{a},$cgi{b},$cgi{c}}' > "$TEST_TEMPLATE"
-testcase 'true' P=text a="ab*bc" b="abbbbbbc"
-testcase '' P=text a="a(^bc)d" b="azd"
-testcase 'true' P=text a="abc.*" b="ABCab\n" c="i" c="m" 
-testcase 'true' P=text a=".*$" b="a\n" c="m"
-testcase 'true' P=text a="^.*$" b="abc\n" c="s"
-testcase 'true' P=text a="   abc#  " b="abc\n" c="x" c="m" c="i"
+testcase 'true' P=text a="ab*c+" b="ac"
+testcase '' P=text a="acb" b="abc"
+testcase 'true' P=text a="[A-Z]bcD" b="abcd" c="i"
+testcase '' P=text a="[A-Z]bcD" b="abcd"
+testcase 'true' P=text a="^abc$" b="def
+abc
+def" c="m"
+testcase '' P=text a="^abc$" b="def
+abc
+def"
+testcase 'true' P=text a="ab(.)*cd" b="ab
+efgh
+cd" c="s"
+testcase '' P=text a="ab(.)*cd" b="ab
+efgh
+cd"
+testcase 'true' P=text a="    #test_comment1
+			[\sab] #test_comment2  " b="de " c="x"
```

Comment:
Won't it be necessary to check whether a space within a character set matches the input string? Because that's one of the main parts of this modifier.
The newlines weren't necessary though I feel now. I'll remove them out.